pkgs/flox/default.nix

```diff
@@ -23,7 +23,9 @@
   fd,
   gnused,
   bats,
-  gitMinimal,
+  git,
```

Comment:
Just highlighting that flox-bash used gitMinimal (the old default.nix contained the string `git = pkgs.gitMinimal`). Any reason we are moving to the larger git package?